src/utils/Terminal.php

```diff
@@ -144,15 +164,25 @@ protected static function getEscapeCodes() : void{
 			self::$COLOR_YELLOW = $colors >= 256 ? $setaf(227) : $setaf(11);
 			self::$COLOR_WHITE = $colors >= 256 ? $setaf(231) : $setaf(15);
 			self::$COLOR_MINECOIN_GOLD = $colors >= 256 ? $setaf(184) : $setaf(11);
+			self::$COLOR_MATERIAL_QUARTZ = $colors >= 256 ? $setaf(188) : $setaf(14);
+			self::$COLOR_MATERIAL_IRON = $colors >= 256 ? $setaf(188) : $setaf(7);
```

Comment:
Are the 256-color codes for these supposed to be the same? This looks suspicious

Reply:
Essentially the same color
![image](https://github.com/pmmp/PocketMine-MP/assets/40174895/4b5df06a-2c7b-4868-b4a4-36c4b34f907c)